src/Orckestra.Composer.Cart/Repositories/Order/OrderRepository.cs

```diff
@@ -195,5 +196,22 @@ public virtual Task<ProcessedCart> ChangeOwnership(ChangeOrderDraftOwnershipPara
 
             return OvertureClient.SendAsync(request);
         }
+
+        public virtual Task<HttpWebResponse> DeleteCartOrderDraft(DeleteCartOrderDraftParam param)
+        {
+            if (param == null) { throw new ArgumentNullException(nameof(param)); }
+            if (param.OrderId == default) { throw new ArgumentException(GetMessageOfEmpty(nameof(param.OrderId))); }
+            if (param.CustomerId == default) { throw new ArgumentException(GetMessageOfEmpty(nameof(param.CustomerId))); }
+            if (param.Scope == default) { throw new ArgumentException(GetMessageOfEmpty(nameof(param.Scope))); }
```

Comment:
We could advance the validation as 
` if (string.IsNullOrWhiteSpace(param.Scope)) { throw new ArgumentException(GetMessageOfNullWhiteSpace(nameof(param.Scope))); }`